migrations/000011_alter_workloads_table_workloadtype.up.sql

```diff
@@ -0,0 +1,9 @@
+ALTER TYPE workloadtype RENAME TO workloadtype_old;
```

Comment:
I was under impression that if we rename the ENUM then table won't be able to reference it or we need to manually change the reference however I can see its updating the name in table as well. So from migration point of view it looks good to me!

Reply:
I have followed this [article](https://dev.to/yogski/dealing-with-enum-type-in-postgresql-1j3g) for reference.